src/app/pages/payment/internationalPayment.component.ts

```diff
@@ -0,0 +1,152 @@
+import { Component, OnInit, OnDestroy, ElementRef, ViewChild } from "@angular/core";
+import { InternationalPaymentService } from "../../services/internationalPayment/international-payment.service";
+import { ToastrService } from "ngx-toastr";
+import { Location } from "@angular/common";
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+  ReactiveFormsModule,
+  AbstractControl
+} from "@angular/forms";
+import { NgxSpinnerService } from "ngx-spinner";
+import { Router } from "@angular/router";
+import { Subscription } from "rxjs";
+import { SafeUrl, DomSanitizer } from '@angular/platform-browser';
+import { removeSubscription } from 'src/app/helpers/unsubscribe';
+
+@Component({
+  selector: "app-payments",
+  templateUrl: "./internationalPayment.component.html",
+  styleUrls: ["./internationalPayment.component.scss"]
+})
+export class PaymentsComponent implements OnInit {
```

Comment:
Since this component handles only international payment, it'd be better to name it appropriately so. maybe `InternationalPayment`. The file name should also be hyphenated, `international-payment.component.ts`. Likewise the associated style and template classes.